.gitattributes

```
@@ -4,3 +4,4 @@
 *.key binary
 *.jar binary
 *.ttf binary
+release-notes-* merge=union
```

Could also match the whole folder with:
```suggestion
**/release-notes/* merge=union
```